content/articles/Developing-simple-Blockchain-app-In-flutter/index.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /developing-simple-blockchain-app-in-flutter/
+title: How to Develop a Simple Blockchain App in Flutter
+description: This article will help the reader understand how to create a simple blockchain application in Flutter.
+author: johnnie-mbugua
+date: 2022-01-07T00:00:00-12:48
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/developing-simple-blockchain-app-in-flutter/hero.jpg
+    alt: Developing Simple Blockchain App in Flutter Hero Image
+---
+Flutter is used to create cross-platform applications that run on iOS, Android, and on the web. It is an advanced framework that supports fast prototyping and high perfomance.
+<!--more--> 
+Flutter utilizes Google's Skia library to draw UI widgets on the screen. In this tutorial, we will be using the `blockchain` library to create a simple Flutter application.
+
+### Prerequisites
+To follow along, you need to:
+- Have Visual Studio Code installed
+- Be familiar with the Flutter frameowrk and Dart language
+
+### Creating a blockchain Flutter application
+To get started, we need to first install the `Flutter SDK`. You can find the installation instructions [here](https://flutter.dev/).
+
+Once you have installed the Flutter SDK, you can create a `new project` by running the following command in your terminal:
+
+```bash
+flutter create my-blockchain-app
+```
+
+Now that our project is created, let's open it in a code editor.  I will be using Visual Studio Code for this tutorial.
+
+When you run the application, you should see a white screen. This is because we have not added any code or widgets to our UI.
+
+### Adding widgets to our UI
```

Comment:
I do not understand how this section is related to the Blockchain app. Add and centering a `Text` widget - what is its connection.

Reply:
for testing purposes

Reply:
Kindly, remove the code and add more blockchain-related content. The Flutter app comes with default widgets for testing if the app is working or not,